cvat-ui/src/reducers/plugins-reducer.ts

```diff
@@ -25,6 +24,17 @@ const defaultState: PluginsState = {
         loginPage: {
             loginForm: [],
         },
+        modelsPage: {
+            topBar: {
+                items: [],
+            },
+            deployedModelItem: {
+                menu: {
+                    items: [],
+                },
+                icon: [],
```

Comment:
What do you mean here? "Icon" does not sound like a place where we can add some components

Reply:
Why? Here I mean plugins in ModelItem icon component which represents its provider

Reply:
I understand what is ``ModelItem``, but ``icon`` looks like thing what we need to integrate, but not the place where we need to integrate

Reply:
Changed to `topBar` integration